Source/MetroXHunter/Inventory/InventoryData.h

```diff
@@ -10,6 +10,7 @@
 
 /**
  * Inventory Data asset
+ * The starting and max amount of the Syringe & ammo of the player.
```

Comment:
Replace with "Data asset representing the initial amounts of the player's inventory."*